pandas/tests/extension/arrow/test_bool.py

```diff
@@ -55,6 +55,10 @@ def test_from_dtype(self, data):
     def test_from_sequence_from_cls(self, data):
         super().test_from_sequence_from_cls(data)
 
+    @pytest.mark.xfail(reason="bad is-na for empty data")
```

Comment:
why is this xfailed?

Reply:
- ArrowBoolArray doesn't accept DataType(null) .
- na_value of BooleanDtype is pyarrow.lib.NullType, so `AttributeError: 'pyarrow.lib.NullType' object has no attribute 'dtype'` in construct_1d_arraylike_from_scalar

Reply:
This seems fine to ignore.